back/index.js

```diff
@@ -1,11 +1,42 @@
+const connection = require('./config/conf');
 const express = require('express');
 const app = express();
 const port = 8000;
 
+
+
+app.use(function(req, res, next) {
+  res.header("Access-Control-Allow-Origin", "*");
+  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+  next();
+});
+
 app.get('/', (request, response) => {
   response.send('Bienvenue sur Express');
 });
```

Comment:
request/response or req/res, regular or arrow function there are the questions :)

Reply:
done